### ISSUE TITLE
PHP version of scraperwiki fails silently when using camel case columns

### ISSUE THREAD

**Issue by wfdd** [openaustralia/morph#1137]
Camel case column names are automatically converted to snake case upon creation of the table using the RedBean API.  Inserts bypass the API and fail as there is no column with the originally-provided camel case name.  This failure is silent for reasons beyond my mental grasp.

**Comment by LoveMyData** [comment]
Current version of RedBean is pretty old, if you have the background with RedBean. Can you put something together and PR on the changes is highly welcome.

**Comment by wfdd** [comment]
Sorry, I don't use PHP with morph - I'd just been asked by an acquaintance to look into why his scraper was failing.

**Comment by chris48s** [comment]
@wfdd - I don't use PHP on morph either, but if I want to use a more recent version of a python package in morph, I can just specify a newer version in my scraper's `requirements.txt` and the more recent package will be used instead of the globally installed one. Can your friend fix his scraper by pulling in a more recent version of the library in his scraper's `composer.json` ?

**Comment by wfdd** [comment]
No, RedBeans is bundled with scraperwiki and has not received an update in 5 years.  Also scraperwiki is the 'blessed' scraping lib here on morph so it should receive bug fixes from upstream.
